codeforamerica/vita-min
# Nj 191 property tax worksheet

## Link to pivotal/JIRA issue
- https://github.com/newjersey/affordability-pm/issues/191
 
## Is PM acceptance required? (delete one)
- Yes - don't merge until JIRA issue is accepted!

**Reminder**: merge main into this branch and get green tests before merging to main
## What was done?
- Replaced concept of "soft no - unsupported" with a screen that prompts the user to fill out a worksheet and tell us the value

## How to test?
- Use Zeus 1 dependent, navigate to the New Jersey Property Tax credit, select Homeowner
- Check the "My main home was subject to property taxes" and the "Did you share ownership of your main home with someone who is not your spouse" checkboxes.
- Verify that you are taken to the Property Tax Worksheet page, rather than the unsupported page
- Enter 10000 for property tax paid
- Validate that XML includes
```
<PropertyTaxDeductOrCredit>
    <TotalPropertyTaxPaid>10000</TotalPropertyTaxPaid>
    <Homeowner>X</Homeowner>
    <PropertyTaxDeduction>10000</PropertyTaxDeduction>
</PropertyTaxDeductOrCredit>
```
- Validate that PDF has 10000 for line 41, property tax deduction and line 40a.
<img width="953" alt="image" src="https://github.com/user-attachments/assets/34079c25-e832-4951-a4f3-afd69a533bed" />

Repeat for each of the different ACs, including renters.

## Screenshots (for visual changes)
- Before - Unsupported
<img width="1006" alt="Pasted Graphic" src="https://github.com/user-attachments/assets/3b76d04a-4be4-41b5-b5cd-790cdf3524ce" />

- After - Worksheet page
<img width="988" alt="image" src="https://github.com/user-attachments/assets/3877baea-2253-4c24-8f7f-f1c84b35a5db" />
